File: src/examples/python/readAsync.py
#!/usr/bin/env python
import rfid

def rfidCallback(message):
	print message

if __name__ == "__main__":

	rfid.init()
	reader = rfid.startReader("tmr:///dev/rfid", rfidCallback)

	_ = raw_input()

	rfid.stopReader(reader)
	rfid.close()

File: src/clients/c++/readAsync.cpp
#include <iostream>
#include "rfid.h"

void myCallback(char* message)
{
	std::cout << "Tag ID: " << message[0] << " RSSI: " << message[1] << std::endl;
}

int main(int argc, char *argv[])
{
	rfid::init();
	rfid::ReaderID readerID = rfid::startReader("tmr:///dev/rfid", myCallback);

	std::cin.get();

	rfid::stopReader(readerID);
	rfid::close();
	return 0;
}
